test/schema.js

```diff
@@ -50,4 +56,170 @@ metaschema.fs.load(null, null, (err, schemas) => {
       const actual = ms.createInstance('schema', {});
       test.strictSame(actual, null);
     });
+
+  const dupSchemas = duplicate(schemas);
+  const [domains] = dupSchemas;
+
+  domains[1].BigInt = { type: 'bigint' };
```

Comment:
Isn't it better to put schemas into a fixtures directory, and load them the way they are supposed to load?